docs/UserGuide.adoc

```diff
@@ -613,6 +618,9 @@ The most recent undo command is reversed.
 Clears all entries from the application. +
 *Format:* `clear`
 
+[NOTE]
+This command clears all the stored information which includes reminders and spending , and also reset the application to the initial state with budget set to $1000.
```

Comment:
This command clears all stored information, including reminders and spending, and sets the budget to a default value of $1000.